resources/specifications.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
This file wont be read. I would suggest moving this into service_guides.json without the categories

Reply:
@mattclark-zerogravit - Noted. See second commit. I don't know if stakeholders will like classifying CTC Traders TIS as a "service guide" but I suppose that it's hard to justify having a category for a single document.

Reply:
@mattclark-zerogravit - Thanks for approval. Will you be able to merge PR today?